weshd/go/shared/shared.go

```diff
@@ -168,7 +168,6 @@ func StartHTTPServer() {
 
 	if err := httpServer.ListenAndServe(); err != nil {
 		log.Println("weshnet: http listenAndServer err", err)
-		panic(errors.Wrap(err, "failed to start http server"))
```

Comment:
It's ok to continue with errors?

Reply:
having it crashed app when disabling chat via toggle switch on android